41_snapshots-btrfs

```diff
@@ -47,6 +53,7 @@ grub_btrfs_config="${sysconfdir}/default/grub-btrfs/config"
 
 [[ -f "$grub_btrfs_config" ]] && . "$grub_btrfs_config"
 . "$datarootdir/grub/grub-mkconfig_lib"
+. "${sysconfdir}/default/grub"
```

Comment:
are you sure this line is needed? I thought we _only_ configure via our own config